packages/sanity/src/core/form/studio/tree-editing/utils/build-tree-editing-state/buildBreadcrumbsState.ts

```diff
@@ -21,36 +14,34 @@ interface BuildBreadcrumbsStateProps {
 export function buildBreadcrumbsState(props: BuildBreadcrumbsStateProps): TreeEditingBreadcrumb {
   const {arraySchemaType, arrayValue, itemPath, parentPath} = props
 
-  const parentTitle = getSchemaTypeTitle(arraySchemaType)
-
   const items: TreeEditingBreadcrumb[] = arrayValue
     .map((arrayItem) => {
       const nestedItemPath = [...parentPath, {_key: arrayItem._key}] as Path
 
       const itemType = getItemType(arraySchemaType, arrayItem)
-
       const isReference = isReferenceSchemaType(itemType)
 
       // Don't add reference items to the breadcrumbs
-      if (isReference) return null
-
-      const {title} = getArrayItemPreview({arrayItem, arraySchemaType})
+      // or items without a type
+      if (isReference || !itemType) return null
```

Comment:
huh is this possible? 🤔 for a schema to not have a type?

Reply:
The `getItemType` function returns `SchemaType | undefined`, so it could potentially be `undefined`. This line is also needed to satisfy TS – without it we would get an error here!

Reply:
Huh! the more you know! 🌈 
Thanks!